src/ngx_http_lua_common.h

```diff
@@ -140,6 +140,8 @@ struct ngx_http_lua_main_conf_s {
 
     ngx_array_t         *shm_zones;  /* of ngx_shm_zone_t* */
 
+    ngx_array_t         *shdict_shm_zones; /* shm zones of "shdict" */
+
```

Comment:
@helloyi I think `shdict_shm_zones` is too verbose. `shdict_zones` is already clear enough since `sh` already means `shm`.